src/core/timeout_middleware.py

```
@@ -52,17 +52,19 @@ async def __call__(
         
         start_time = time.time()
-        logger.debug(f"Starting tool '{tool_name}' (ID: {operation_id}) with {timeout_seconds}s timeout")
+        logger.info(f"🚀 TIMEOUT MIDDLEWARE: Starting tool '{tool_name}' (ID: {operation_id}) with {timeout_seconds}s timeout")
+        logger.info(f"🔍 Request method: {getattr(request, 'method', 'unknown')}")
         
         try:
+            logger.info(f"⏳ TIMEOUT MIDDLEWARE: Calling next handler for '{tool_name}'...")
             # Execute the tool with timeout protection
             result = await asyncio.wait_for(
                 call_next(request), 
                 timeout=timeout_seconds
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)